.travis.yml

```diff
@@ -20,7 +20,7 @@ before_script:
   - "sudo sed -i 's/^127\\.0\\.0\\.1.*$/127.0.0.1 localhost/' /etc/hosts"
 
     # Lets know what CouchDB we are playing with
-  - "curl -X GET http://127.0.0.1:5984/"
+  - "curl -X GET http://127.0.0.1:5984/" # should this use COUCH_HOST variable?
```

Comment:
COUCH_HOST is defined as part of the test scripts so wouldnt be picked up here, this was mostly for debugging previous issues so not entirely needed anymore